WordPress/src/main/res/layout/stats_block_list_action_card.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Do we need to use card view here? If I'm not mistaken, this card view is inside another card view (`stats_list_block`) and that causes two-card view strokes on the screen.